department-of-veterans-affairs/caseflow
# Date selector

Connects #5819 

### Description
This PR adds a date selector to the upload page.

### Testing Plan
1. Go to 'schedule/build/upload'
1. Select either 'RO/CO hearings' or 'Judge non-availability' 
1. Confirm that a date selector is displayed
1. Confirm that when editing the dates, the redux store is updated
